cypress/integration/accessibility_PageHeader_spec.js

```diff
@@ -12,6 +12,10 @@ describe('PageHeader Accessibility check', () => {
           id: 'heading-order',
           enabled: false,
         },
+        {
+          id: 'color-contrast',
```

Comment:
Why does this rule need to be disabled? Can we document that reason in the code?

Reply:
<img width="1128" alt="Screen Shot 2022-04-04 at 7 28 39 PM" src="https://user-images.githubusercontent.com/10593890/161648799-587ef4ea-6143-4f70-96ab-e33234d523ed.png">